Library/Formula/rsync.rb

```diff
@@ -0,0 +1,1385 @@
+class Rsync < Formula
+  desc "Utility that provides fast incremental file transfer"
+  homepage "https://rsync.samba.org/"
+  url "https://rsync.samba.org/ftp/rsync/rsync-3.2.7.tar.gz"
+  mirror "https://mirrors.kernel.org/gentoo/distfiles/rsync-3.2.7.tar.gz"
+  mirror "https://www.mirrorservice.org/sites/rsync.samba.org/rsync-3.2.7.tar.gz"
+  sha256 "4e7d9d3f6ed10878c58c5fb724a67dacf4b6aac7340b13e488fb2dc41346f2bb"
+  license "GPL-3.0-or-later"
+
+  depends_on "lz4"
+  depends_on "openssl"
+  depends_on "popt"
+  depends_on "xxhash"
+  depends_on "zlib"
+
+  def install
+    ENV.append_to_cflags "-DSUPPORT_FILEFLAGS"
+    args = %W[
+      --prefix=#{HOMEBREW_PREFIX}
```

Comment:
Can you make this `prefix` instead? This is installing rsync in a way where it's not being managed by Tigerbrew. Unfortunately we don't have the sandboxing setup to actively prevent that right now.

Reply:
Sure, don.